docs/documentation-styleguide.md

```diff
@@ -0,0 +1,185 @@
+
+The [Gatsby MDX reference guide](https://www.gatsbyjs.org/docs/mdx/) contains helpful information for writing pages in MDX.
+
+#### Release Notes and Changelog
+Historically, releasing a new version of Carbon was a manual process. The release notes were written and updated by each collaborator manually in the old `CHANGELOG.md` file. The significant number of changes from `v8` to `v9` due to not releasing often enough have made it difficult for users to upgrade. As a result, we moved to use [`semantic-release`](https://github.com/semantic-release/semantic-release) to allow us to release frequently.
+
+Depending on the type of commit to the `master` branch, each subsequent successful build now automatically publishes a new version of Carbon in accordance with [Semantic Versioning](https://semver.org), generates the changelog and the release notes. Enter a succinct commit message in the correct format described in the [`semantic-release.md`](../rfcs/text/semantic-release.md) RFC and [`semantic-release`](https://github.com/semantic-release/semantic-release) will complete the process for you.
+
+#### Request For Change (RFC)
+When you want to propose a substantial change to Carbon, such as the addition of a new component or a breaking change, you need to submit an RFC. The core Carbon Team will discuss the RFC needs to incorporate design constraints and reach a consensus before it is progressed.
+
+There is a [template](../rfcs/template.md) to guide you when writing an RFC. You can add other sections if appropriate. Use hyphens to separate words in the file name and save it as a `.md` file in the `carbon/rfcs/text` directory. If you need to add supporting images, save them in the `carbon/rfcs/images` directory.
+
+#### README files
+We write README files in MD format to tell collaborators about what Carbon and [Cypress.io](https://www.cypress.io) are, how collaborators can get started and where they can find further information or assistance.
+
+The [Carbon README](../README.md) contains:
+- a link to a more detailed 'Getting Started' guide;
+- a link to the [contributing guide](../CONTRIBUTING.md) explaing how to contribute to the project; and
+- license information.
+
+The [Cypress README](../cypress/README.md) contains:
+- instructions of how to install and run Cypress;
+- how to access test results; and
+- a link to the [testing styleguide](testing-styleguide.md) where a detailed explanation of how to write Cypress tests can be found.
+
+The [RFCs README](../rfcs/README.md) provides detailed instructions of when and how to write an RFC. 
+
+All README files are accessible through GitHub from their respective directories.
+
+When writing a README file, consider:
+- who your audience is;
+- what useful information you want in there;
+- if a table of contents is required;
+- the [use of links](#use-of-links); and
```

Comment:
```suggestion
- the [use of links](#use-of-links);
```

Reply:
I followed the first example in "A list of very short points" in this guide for bullet point lists: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html. However, if you think the second example without an "and" before the last point, then I'm happy to change.

Reply:
Removed the "and" in all lists in this document.